### PR TITLE
tests(js): Convert Performance Context to RTL

### PR DESCRIPTION
**Pull request by k-fish**
This converts the performance content test to RTL, and moving it to TS as well. This should be the last RTL test for the perf directory.

### CODE REVIEW

File: static/app/views/performance/content.spec.tsx [hunk 1]
```diff
@@ -46,6 +45,7 @@ function initializeData(projects, query, features = FEATURES) {
     projects,
   });
   const initialData = initializeOrg({
+    ...initializeOrg(),
```
Comment (gggritso) [inline static/app/views/performance/content.spec.tsx:48]: Oh, what's this? It looks redundant

**Review by gggritso — APPROVED** [review]
Very nice!

File: static/app/views/performance/content.spec.tsx (hunk 1 above)
Comment (k-fish) [reply static/app/views/performance/content.spec.tsx:48]: fixing for ts with essentially defaults + override, saw this trick elsewhere and yoinked it. Free project stubs! 😆

Comment (gggritso) [reply static/app/views/performance/content.spec.tsx:48]: Interesting!

**Pull request merged**